clients/dashboards/dashboards.go

```diff
@@ -0,0 +1,66 @@
+package dashboards
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.DashboardsApi
+}
+
+type Params struct {
+	clients.OrgParams
+	Ids []string
+}
+
+func (c Client) List(ctx context.Context, params *Params) error {
+	if !params.OrgID.Valid() && params.OrgName == "" && c.ActiveConfig.Org == "" && len(params.Ids) == 0 {
+		return fmt.Errorf("at least one of org, org-id, or id must be provided")
+	}
+
+	const limit = 100
+	req := c.GetDashboards(ctx)
+	req = req.Limit(limit)
+	if params.OrgID.Valid() {
+		req = req.OrgID(params.OrgID.String())
+	}
+	if params.OrgName != "" {
+		req = req.Org(params.OrgName)
+	}
+	if !params.OrgID.Valid() && params.OrgName == "" {
+		req = req.Org(c.ActiveConfig.Org)
+	}
+	orgID := req.GetOrgID() // for logging purposes
```

Comment:
This will be an empty string if `--org-id` wasn't passed. I think you can drop it and simplify the error message slightly, see suggestion below